setup.py

```diff
@@ -15,8 +15,8 @@
     "xarray",
 ]
 
-test_requirements = ["pytest", "pytest-subtests", "serialbox"]
-ndsl_requirements = ["ndsl @ git+https://github.com/NOAA-GFDL/NDSL.git@2024.04.00"]
+test_requirements = ["pytest", "pytest-subtests", "serialbox", "coverage"]
+ndsl_requirements = ["ndsl @ git+https://github.com/NOAA-GFDL/NDSL.git@develop"]
```

Comment:
quick question, why the develop branch and not a specific tag?

Reply:
We are using the `develop` branch as the release is not ready yet, and that the `develop` branch contains the configurations needed for testing. Once the orchestration issue is resolved the new release will be made and there will be a subsequent PR to update this.